charts/hotrod/values.yaml

```
@@ -0,0 +1,177 @@
+  #   port: http
+# -- Node selector for hotrod
+nodeSelector: {}
+# -- Tolerations for hotrod
+tolerations: []
+# -- Affinity for hotrod
+affinity: {}
+#########################
+# Locust Deployment     #
+#########################
+locust:
+  # -- Whether to enable locust 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)